modules/client-configuration/pages/client-upgrades-uyuni.adoc

```diff
@@ -24,19 +24,19 @@ opensuse_leap15_2-uyuni-client
 // These are custom channels.
 // For more information, see xref:client-configuration:clients-opensuse.adoc[].
 
-. In the {productname} Server {webui}, navigate to menu:Software[Manage > Channels] and click the [systemitem]``Uyuni Client Tools for openSUSE Leap 15.2 (x86_64)`` channel name.
+. In the {productname} Server {webui}, navigate to menu:Software[Manage > Channels] and click the [systemitem]``Uyuni Client Tools for openSUSE Leap 15.4 (x86_64)`` channel name.
```

Comment:
And probably replace a lot of ocurrences with the variable for the version. But not for this PR :-P